phpBB/phpbb/db/migration/data/v310/style_update_p1.php

```diff
@@ -69,10 +69,11 @@ public function styles_update()
 		$skip_dirs = array('.', '..', 'prosilver');
 		foreach ($iterator as $fileinfo)
 		{
-			if ($fileinfo->isDir() && !in_array($fileinfo->getFilename(), $skip_dirs) && file_exists($fileinfo->getPathname() . '/style.cfg'))
+			if ($fileinfo->isDir() && !in_array($fileinfo->getFilename(), $skip_dirs) && file_exists($fileinfo->getPathname() . '/composer.json'))
```

Comment:
what will happen in case of update from 3.0.13 if their the installed styles are not 3.2 compatible?